index.html

```diff
@@ -4,10 +4,123 @@
+
+      <div class="menu-burger-icon">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+    </nav>
+
+    <section >
+      
+      <div class="img ecllipse1-img">
+        <img src="./assets/images/Ellipse-3.png" alt="Elipse">
+      </div>
+
+      <div class="img block1-img">
+        <img src="./assets/images/Rectangle-4.png" alt="Elipse">
+      </div>
+
+      <div class="img block2-img">
+        <img src="./assets/images/Rectangle-6.png" alt="Elipse">
+      </div>
+
+      <div class="img ecllipse2-img">
+        <img src="./assets/images/Ellipse-11.png" alt="Elipse">
+      </div>
+
+      <div class="img block3-img">
+        <img src="./assets/images/Rectangle-7.png" alt="Elipse">
+      </div>
+
+      <div class="img circle1-img">
+        <img src="./assets/images/Ellipse-5.png" alt="Elipse">
+      </div>
+
+      <div class="img circle2-img">
+        <img src="./assets/images/Ellipse-8.png" alt="Elipse">
+      </div>
+
+      <div class="img circle3-img">
+        <img src="./assets/images/Ellipse-14.png" alt="Elipse">
+      </div>
+      <div class="img circle4-img">
+        <img src="./assets/images/Ellipse-12.png" alt="Elipse">
+      </div>
+
+      <div class="img ecllipse3-img">
+        <img src="./assets/images/Ellipse-13.png" alt="Elipse">
+      </div>
+      
+      <div class="img block4-img">
+        <img src="./assets/images/Rectangle-5.png" alt="Elipse">
+      </div>
+
+      <div class="img circle5-img">
+        <img src="./assets/images/Ellipse-4.png" alt="Elipse">
+      </div>
+      
+      <div class="img ecllipse4-img">
+        <img src="./assets/images/Ellipse-15.png" alt="Elipse">
+      </div>
+
+      <div class="about-me">
+        <div class="title-container">
+          <h2 class="welcome-txt">I am Mhamad, Glad to see you!</h2>
+        </div>
+        
+        <div class="description-bg">
+
+          <div class="description">
+            I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded,
+             don’t hestiate to contact me.
+          </div>
+
+        </div>
+
+        <section class="contact-me">
+        
+          <div class="contact-me-title">
+            <h2>LET’S CONNECT</h2>
+          </div>
+          
+          <div class="contact-me-icons ">
+              <a href="https://www.linkedin.com/in/mhamad-raad-446a75227/" target="_blank" rel="noopener">
+                <img src="./assets/images/linkedin.png" class="linkedin-img icon" alt="linkedin">
+              </a>   
+              
+              <a href="https://github.com/Mhamad-Raad/" target="_blank" rel="noopener">
+                <img src="./assets/images/Vector-4.png" class="github-img icon" alt="Github">
+              </a> 
+                
+          </div>
```

Comment:
- [ ] Kindly add all icons that are present in Figma template. Also, the icons should be wrapped in a `ul` element instead of `div`. Each icon should be an `a` tag wrapped in a `li` element. this way you can link your social networks to them via the `href` attribute 💯 

[Optional]
You can use [font awesome](https://fontawesome.com/) for the icons instead of images